bin/check_python.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
I am sorry to be so annoying with this, but could you add this `if` between these two lines:
```bash
if [ "$(ps -o comm= -p $$)" = "zsh" ]; then
    THIS_DIR="$( cd "$( dirname "${ZSH_ARGZERO}" )" >/dev/null 2>&1 && pwd )"
fi
```